backend/test/v2/integration/pipeline_api_test.go

```diff
@@ -160,7 +160,7 @@ func (s *PipelineApiTest) TestPipelineAPI() {
 		&params.PipelineServiceCreatePipelineVersionParams{
 			PipelineID: argumentUrlPipeline.PipelineID,
 			Body: &model.V2beta1PipelineVersion{
-				DisplayName: "argumentUrl-v1",
+				DisplayName: "argumenturl-v1",
```

Comment:
Why was this change needed?

Reply:
I take it it's because display name isn't stored in the database and `pipelineAPIToModelFieldMap` shows that name and display name are the same, so it has to be a valid Kubernetes name.

Reply:
Correct, unless we make them independent and treat the fields according to the storage type.